lib/pbench/server/api/resources/query_apis/controllers_list.py

```diff
@@ -58,9 +58,9 @@ def assemble(self, json_data: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
             "start" and "end" are datetime objects representing a set of Elasticsearch
                 run document indices in which to search.
         """
-        user = json_data["user"]
-        start = json_data["start"]
-        end = json_data["end"]
+        user = json_data.get("user")
+        start = json_data.get("start")
+        end = json_data.get("end")
```

Comment:
The change from array form to `.get` isn't necessary here since we know they'll exist; but, sure, the consistency doesn't hurt either since missing parameters will have already been validated.

Reply:
yes I changed these for the consistency